assets/dev/scss/editor/preview/preview.scss

```diff
@@ -1078,6 +1078,12 @@ html.elementor-html {
 						fill: $editor-dark;
 					}
 				}
+
+				&[data-preset="r100"] {
+					@if 'rtl' == $direction {
+						transform: scaleX(-1);
+					}
+				}
```

Comment:
I am happy with this change. 

I wander if we couldn't replace the SVG 'snippets' inside the php code with standard Elementor Eicons. 

@StyleShit : can you please comment on this aspect?

Reply:
Evyatar has approved this approach.